src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todos } from '../../interfaces/Todos';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todos;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      key={todo.id}
```

Comment:
please remove the redundant key, you already set the key in the `TodoList` component